### ISSUE TITLE
Screen is not display when trying stream iOS

### ISSUE THREAD

**Issue by mhaqqiw** [NetrisTV/ws-scrcpy#110]
error log

> `[QvhackRunner][udid: 4fd3237e29cbf44aed90be373df4df4d937482f8] "exit" event. code null, signal: SIGTERM
[QvhackRunner][udid: 4fd3237e29cbf44aed90be373df4df4d937482f8] "close" event. code null, signal: SIGTERM`

**Comment by drauggres** [comment]
Hi. Can you verify that QT and qvh work with your device (https://github.com/NetrisTV/ws-qvh#steps-to-set-up)?

**Comment by krishtoautomate** [comment]
i think, this is qvh issue and not related to this project.

**Comment by inluxc** [comment]
I'am also having the same problem. i added a debug of the address.

```
ws://127.0.0.1:8001/ws?stream=00008030001550A4029A402E%00%00%00%00%00%00%00%00%00%00%00%00%00%00%00%00
[QvhackRunner][udid: 00008030001550A4029A402E] "exit" event. code null, signal: SIGTERM
[QvhackRunner][udid: 00008030001550A4029A402E] "close" event. code null, signal: SIGTERM
```

**Comment by drauggres** [comment]
> Hi. Can you verify that QT and qvh work with your device (https://github.com/NetrisTV/ws-qvh#steps-to-set-up)?

Same question.

**Comment by inluxc** [comment]
Yep i can connect with quicktime to the iphone and i can record with qvh to file.
But with ws-qvh, i can connect once in 100 tries.

**Comment by inluxc** [comment]
I cheking that this message is showing in the browser console:

[StreamReceiver]. WS closed: connect ECONNREFUSED 127.0.0.1:8001
